ai_diffusion/client.py

```diff
@@ -195,6 +189,23 @@ async def connect(url=default_url):
         _ensure_supported_style(client)
         return client
 
+    def get_node_optional_values(self, node_name: str, args: dict | None = None) -> dict[str, Any]:
+        if self.nodes_required_inputs.get(node_name, None) is None:
```

Comment:
could put the result in a variable here and reuse it below (avoid a 2nd dict lookup)